playbooks/roles/runner-setup/tasks/assert_packages.yml

```diff
@@ -0,0 +1,67 @@
+---
+#
+# (c) Copyright 2019 SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+- name: Configure repositories for skuba and terraforms
+  become: yes
+  zypper_repository:
+    name: "{{ caasp_pkgs_repo_version }}"
+    repo: "{{ socok8s_repos_to_configure[caasp_pkgs_repo_version] }}"
+    autorefresh: True
+    auto_import_keys: yes
+    state: present
+  when:
```

Comment:
you can remove the condition, as this is always defined.

Reply:
@evrardjp : This condition is important as we don't want to do anything if repos are not defined in common-vars.yml for that OS version e.g. opensuse leap or some other OS ?

Reply:
I approved this as I made a follow up PR to fix my eventual concerns.